frontend/src/utility.ts

```diff
@@ -2,6 +2,8 @@ import _ from "lodash";
 import moment from "moment";
 import { ICount } from "@/types/DataTable";
 
+export const filteredStatus = ["Lost", "Opted out", "Incorrect", "Duplicate"];
+
```

Comment:
This statuses are actually belonging to the users only. So why not to move it into `UsersByRole`?